openedx/core/djangoapps/programs/tests/test_utils.py

```diff
@@ -113,6 +113,37 @@ def test_single_program_engagement(self, mock_get_programs):
         )
         self.assertEqual(meter.completed_programs, [])
 
+    def test_count_only_false(self, mock_get_programs):
+        """
+        Verify that the progress meter returns the program's courses in the correct
+        format if the count_only flag is False.
+        """
+        course_run_key = generate_course_run_key()
+        data = [
+            ProgramFactory(
+                courses=[
+                    CourseFactory(course_runs=[
+                        CourseRunFactory(key=course_run_key),
+                    ]),
+                ]
+            )
+        ]
+        mock_get_programs.return_value = data
+
+        self._create_enrollments(course_run_key)
+
+        meter = ProgramProgressMeter(self.user)
+
+        program = data[0]
+        expected = ProgressFactory(
```

Comment:
@MatthewPiatetsky yeah. It looks kind of silly here, but it cuts down on duplicate code elsewhere (when you only want to change one key, for example).